lib/iris/experimental/ugrid/__init__.py

```diff
@@ -8,4029 +8,29 @@
 Infra-structure for unstructured mesh support, based on
 CF UGRID Conventions (v1.0), https://ugrid-conventions.github.io/ugrid-conventions/
 
+.. note::
+
+    For the docstring of :const:`PARSE_UGRID_ON_LOAD`: see the original
+    definition at :const:`iris.experimental.ugrid.load.PARSE_UGRID_ON_LOAD`.
+
 """
+# Configure the logger.
+from ...config import get_logger
 
-from abc import ABC, abstractmethod
-from collections import namedtuple
-from collections.abc import Iterable
-from contextlib import contextmanager
-from functools import wraps
-from itertools import groupby
-import logging
-from pathlib import Path
-import re
-import threading
+logger = get_logger(__name__, fmt="[%(cls)s.%(funcName)s]")
 
-import dask.array as da
-import numpy as np
+# Lazy imports to avoid circularity.
+from .load import load_mesh, load_meshes, PARSE_UGRID_ON_LOAD  # isort:skip
+from .mesh import Connectivity, Mesh, MeshCoord  # isort:skip
+from .save import save_mesh  # isort:skip
 
-from ... import _lazy_data as _lazy
-from ...common.lenient import _lenient_service as lenient_service
-from ...common.metadata import (
-    SERVICES,
-    SERVICES_COMBINE,
-    SERVICES_DIFFERENCE,
-    SERVICES_EQUAL,
-    BaseMetadata,
-    metadata_filter,
-    metadata_manager_factory,
-)
-from ...common.mixin import CFVariableMixin
-from ...config import get_logger
-from ...coords import AuxCoord, _DimensionalMetadata
-from ...exceptions import ConnectivityNotFoundError, CoordinateNotFoundError
-from ...fileformats import cf, netcdf
-from ...fileformats._nc_load_rules.helpers import get_attr_units, get_names
-from ...io import decode_uri, expand_filespecs
-from ...util import guess_coord_axis
 
 __all__ = [
     "Connectivity",
-    "ConnectivityMetadata",
     "load_mesh",
     "load_meshes",
     "save_mesh",
```

Comment:
@trexfeathers You've inherited this... but I *think* we're sorting alphabetical (exclusive of case), so `save_mesh` should be last, right?

Reply:
c71b88751da2d0c73ebf482ddc4e9ef6127a1934